js/interface.js

```diff
@@ -1411,17 +1496,7 @@ $('.file-manager-wrapper')
     }
   })
   .on('click', '[data-browse-trash]', function() {
-    $('[data-browse-trash] span').addClass('active-trash');
-    $('[restore-action]').show(); 
-    $('[file-remove-trash]').show();
-
-    $('[rename-action]').hide();
-    $('[delete-action]').hide();
-
-    disableSearchState();
-    resetUpTo($(this));
-    getFolderContents($(this), true);
-    updateSearchTypeOptions($(this).data('type'));
+    loadTrashFolder($(this));
```

Comment:
So far you have only fed `$('[data-browser-trash]')` to the function. There's no need for a parameter.

```suggestion
    loadTrashFolder();
```

Reply:
@tonytlwu, i need to pass the `$('[data-browse-trash]')` because in other way the page is not loading

Reply:
Structurally, this is what I'm seeing in your code.

```js
function foo(param) {
  console.log(param);
}

foo(1);
foo(1);
```

You're passing the same parameter every single time. Let me know if there's any instance that's calling it using a different parameter. Otherwise, you can update it to.

```js
function foo() {
  console.log(1);
}

foo();
foo();
```